### PR TITLE
(Docs): Updated various fields in the p5.AudioIn reference

### LINKED ISSUE #724: Docs : Update various fields in the `p5.AudioIn` reference
In the `p5.AudioIn` reference, description of some fields are not provided like: `input`, `output`, `stream`, etc, but information about these fields are available by clicking on those respected links, but not specified for the AudioIn reference unlike fields like `embed` and `amplitude`. 

decided to update specific fields like `currentSource`, `mediaStream` etc for now and skip common fields like `input`, `output` etc as discussed. 

refer : https://p5js.org/reference/#/p5.AudioIn

image (2)

### PR DESCRIPTION
**Pull request by Abhijay007**
Signed-off-by: Abhijay Jain <Abhijay007j@gmail.com>

added description for the various fields of `p5.AudioIn reference` such as `stream`, `mediaStream` and `currentSource`

resolves #724

### CODE REVIEW

**Comment by Abhijay007** [comment]
cc :  @davepagurek

File: src/audioin.js [hunk 1]
```diff
@@ -50,30 +50,45 @@ p5sound.inputSources = [];
...
+     * It creates a MediaStream object that can be used to start and stop the mic and measure its volume using the getLevel() method or by connecting it to an FFT object.
+     * MediaStream object can also be use to check if the browser supports MediaStreamTrack and mediaDevices and to add the AudioIn object to the soundArray for disposal on close.
+     *
      * @property {MediaStreamAudioSourceNode|null} mediaStream
      */
     this.mediaStream = null;
-    
+
     /**
+     * use to store the "current source of audio input", such as the user's microphone.
```
Comment (davepagurek) [inline src/audioin.js:88]: I think this one should be "Used" for consistency

File: src/audioin.js [hunk 2]
```diff
@@ -50,30 +50,45 @@ p5sound.inputSources = [];
...
+     * initially set to "null" and can be updated as the user selects different audio sources.
+     * also used in conjunction with the "input" and "mediaStream" properties to control audio input.
```
Comment (davepagurek) [inline src/audioin.js:90]: Capitalize "Also" here too

**Review by davepagurek — COMMENTED** [review]
Thanks for documenting these, the writing's great! Just minor comments again for come verb tense consistency, and also can we capitalize the start of all the sentences again? Thanks!

**Review by davepagurek — APPROVED** [review]
Awesome!

**Pull request merged**